templates/shared/_ubuntu-advantage-service-description.html

```diff
@@ -0,0 +1,68 @@
+  <div class="u-fixed-width">
+    <div style="overflow-x: auto;">
+      <table class="p-table" style="width: auto;">
+        <thead>
+          <tr>
+            <th style="width: 40%;"></th>
+            <th width="200" class="u-align--center">Essential</th>
+            <th width="200" class="u-align--center">Standard</th>
+            <th width="200" class="u-align--center">Advanced</th>
+          </tr>
+        </thead>
+        <tbody>
+          <tr>
+            <td>Support</td>
+            <td class="u-align--center">No</td>
+            <td class="u-align--center">24/5</td>
+            <td class="u-align--center">24/7</td>
+          </tr>
+          <tr>
+            <td>Severity 1 response time</td>
+            <td class="u-align--center">No</td>
+            <td class="u-align--center">4h</td>
+            <td class="u-align--center">1h</td>
+          </tr>
+          <tr>
+            <td><a href="/legal/ubuntu-advantage-assurance">Ubuntu assurance program</a></td>
+            <td class="u-align--center">No</td>
+            <td class="u-align--center">{{ image(url="https://assets.ubuntu.com/v1/f1a7515d-tick-darkaubergine.svg", alt="Yes", width="16", height="16", hi_def=True, loading="lazy",) | safe }}</td>
+            <td class="u-align--center">{{ image(url="https://assets.ubuntu.com/v1/f1a7515d-tick-darkaubergine.svg", alt="Yes", width="16", height="16", hi_def=True, loading="lazy",) | safe }}</td>
+          </tr>
+          <tr>
+            <td><a href="/esm">ESM</a></td>
```

Comment:
Shouldn't this be "Extended Security Maintenance (ESM)"?

Reply:
it should! thanks